gui-apps/grim/grim-1.2.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="Grab images from a Wayland compositor."
+HOMEPAGE="https://github.com/emersion/grim"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/emersion/${PN}.git"
+else
+	SRC_URI="https://github.com/emersion/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="+man jpeg"
```

Comment:
Are you sure you want this `+` in here, explicitly making users who don't want man pages  installed (with exta build deps) use package.use for this package?
This might be a personal thing and I'd be fine with it if it build man pages without extra deps.

I saw you had `+` in your other new packages as well, whatever you decide please reflect it on the other PRs as well.

Reply:
I feel like not wanting man-pages is rather rare, and in the case where that is truly desired it will be applied system-wide with `-man` on make.conf. 

I can, of course, change this if not wanting man pages is not as unusual as I anticipated :)

Reply:
The man pages require app-text/scdoc which is a dependency.